### PR TITLE
Update exiled.json

### PR DESCRIPTION
**Pull request by Yzzuwyzzuf**
## About The Pull Request

when your cats kill each other, what's the usual decision? exile, right? and then to the dark forest for them? well why aren't there thoughts for that? any cat would hold a grudge so that's what I've decided to add!

## Why This Is Good For ClanGen

as explained above, its something people would see pretty often and would make the dark forest feel more fleshed out

## Linked Issues

none

## Changelog/Credits

added thoughts for exiled dark forest cats- Fuzzy

### CODE REVIEW

File: resources/dicts/thoughts/dead/darkforest/exiled.json [hunk 1]
```diff
@@ -1,3 +1,63 @@
 [
-    
-]
\ No newline at end of file
+  {
+    "id": "exiled_df_dead_thoughts",
+    "thoughts": [
+      "Is thinking of the dissaproving glares of {PRONOUN/m_c/poss} old Clanmates when they exiled {PRONOUN/m_c/object} while {PRONOUN/m_c/subject} train",
```
Comment (keplerstelescope) [inline resources/dicts/thoughts/dead/darkforest/exiled.json]:
```suggestion
      "Is thinking of the disapproving glares of {PRONOUN/m_c/poss} ex-Clanmates when they exiled {PRONOUN/m_c/object} while {PRONOUN/m_c/subject} {VERB/m_c/train/trains}",
```
sentence feels a little meandering, maybe something like `Thinks of the disapproving glares of {PRONOUN/m_c/poss} ex-Clanmates while {PRONOUN/m_c/subject} {VERB/m_c/train/trains}` might be better?

File: resources/dicts/thoughts/dead/darkforest/exiled.json [hunk 2]
```diff
@@ -1,3 +1,63 @@
...
+        "Is watching {PRONOUN/m_c/poss} old Clan with scorn hoping for an opportunity to get back at them",
```
Comment (keplerstelescope) [inline resources/dicts/thoughts/dead/darkforest/exiled.json]:
```suggestion
        "Is watching {PRONOUN/m_c/poss} old Clan with scorn, hoping for an opportunity to get back at them",
```

File: resources/dicts/thoughts/dead/darkforest/exiled.json [hunk 3]
```diff
@@ -1,3 +1,63 @@
...
+        "Wants to make {PRONOUN/m_c/poss} old leader pay for exiling {PRONOUN/m_c/object}",
+        "Is planning on making c_n pay for exiling {PRONOUN/m_c/object}"
+        ]
+    },
+    {
+        "id": "exiled_df_evil",
+        "thoughts": [
+            "Is imagining the tree {PRONOUN/m_c/subject}{VERB/m_c/'re/'s} clawing at is an old Clanmate",
```
Comment (keplerstelescope) [inline resources/dicts/thoughts/dead/darkforest/exiled.json]:
```suggestion
            "Is imagining the tree {PRONOUN/m_c/subject}{VERB/m_c/'re/'s} clawing at is one of {PRONOUN/m_c/poss} ex-Clanmates",
```
edited wording to sound slightly better. "clawing up" might also be better than "clawing at"?

File: resources/dicts/thoughts/dead/darkforest/exiled.json [hunk 4]
```diff
@@ -1,3 +1,63 @@
...
+            "Trains extra hard waiting for an opportunity to get {PRONOUN/m_c/poss} revenge",
```
Comment (keplerstelescope) [inline resources/dicts/thoughts/dead/darkforest/exiled.json]:
```suggestion
            "Trains extra hard, waiting for an opportunity to get {PRONOUN/m_c/poss} revenge",
```
this one is a tentative suggestion, either way works okay

File: resources/dicts/thoughts/dead/darkforest/exiled.json [hunk 5]
```diff
@@ -1,3 +1,63 @@
...
+            "Is deeply regretting what {PRONOUN/m_c/subject} did to get exiled and end up here",
+            "Is reflecting on what {PRONOUN/m_c/subject} did and why {PRONOUN/m_c/subject} ended up in the Dark Forest"
+        ],
+    "main_trait_constraint": [
+        "compassionate",
+        "faithful",
+        "loving",
+        "patient",
+        "thoughtful",
+        "Insecure",
```
Comment (keplerstelescope) [inline resources/dicts/thoughts/dead/darkforest/exiled.json]: insecure is actually closer to a mean trait; its aggressiveness requirement is high and it tends to lead cats with the trait to being more hostile or violent. maybe replace it with the nervous trait here?

File: resources/dicts/thoughts/dead/darkforest/exiled.json [hunk 6]
```diff
@@ -1,3 +1,63 @@
...
+        "wise"
```
Comment (keplerstelescope) [inline resources/dicts/thoughts/dead/darkforest/exiled.json]:
```suggestion
        "wise",
        "sincere"
```

**Review by keplerstelescope — COMMENTED** [review]
first passover! (if i missed anything obvious it's nearly midnight and i'm sorry -w-)

File: resources/dicts/thoughts/dead/darkforest/exiled.json (hunk 5 above)
Comment (Yzzuwyzzuf) [reply resources/dicts/thoughts/dead/darkforest/exiled.json]: good point, will swap that out now

File: resources/dicts/thoughts/dead/darkforest/exiled.json (hunk 1 above)
Comment (Yzzuwyzzuf) [reply resources/dicts/thoughts/dead/darkforest/exiled.json]: definitely much more concise, ty

File: resources/dicts/thoughts/dead/darkforest/exiled.json (hunk 5 above)
Comment (Yzzuwyzzuf) [reply resources/dicts/thoughts/dead/darkforest/exiled.json]: it is done :)

File: resources/dicts/thoughts/dead/darkforest/exiled.json (hunk 1 above)
Comment (Yzzuwyzzuf) [reply resources/dicts/thoughts/dead/darkforest/exiled.json]: added the suggested wording

**Review by scribblecrumb — APPROVED** [review]

**Pull request merged**